re/brokers/brokers_nats.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !rabbitmq
+// +build !rabbitmq
+
+package brokers
```

Comment:
Would it be possible to move this to Writer?
Also, could we add `Publisher` for both `Writer` and `Alarms`, so that RE only have only `Publisher`?